features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,11 @@
+When("I visit the site") do
+    visit root_path
+end
+
+And("show me the page") do
+    save_and_open_page
+  end
```

Comment:
Please format your code.

Reply:
But also, using Selenium and Chromedriver is a much better way...

Reply:
Done and added.

Reply:
Done that now.

Reply:
Done, added.